.circleci/config.yml.d/amb_oss.yml

```diff
@@ -379,10 +393,18 @@ workflows:
             branches:
               only:
                 - master
+      - schedule:
+          # Time is in UTC: 1AM EDT every day
+          cron: "0 5 * * *"
+          filters:
+            branches:
+              only:
+                - shared/emissary
```

Comment:
change to master?

Reply:
Not yet, I don't think? I want nightlies on `shared/emissary`... 🙂